File: 20.03.25/ps.js
let obj ={}
console.log(obj);
let arr =[2,2,2,4,5,5,6];
for(let i=0;i<=arr.length-1;i++){
    if(obj[arr[i]]===undefined|| obj[arr[i]]===0){
        obj[arr[i]] =1;
        console.log(obj)
    }
    else{
      obj[arr[i]]++;
    }
}
let max=0;
let maxelement;
for(let key in obj){
   if(obj[key]>max){
    max=obj[key];
    console.log(max);
    maxelement= key;
    console.log(key);
   }
}
console.log(max,maxelement);

File: 20.03.25/ps.py
obj={}
arr=[22,23,32,32,32,22,23]

for num in arr:
    if num in obj:
        obj[num]+=1
    else:
        obj[num]=1

print(obj.items())
maxf=0
maxc=None

for key ,val in obj.items():
    if val > maxf:
        maxf=val
        maxc=key

print(key)
        

